Binarize tags finding improved

denniscfeng opened the pull request:
New tags finding method should reduce remaining failure cases when tags aren't found properly
- Binarized HSV of butterfly is added to binarized tags to get all relevant regions of image (also used hole filling and erosion)
- Butterfly is then removed, remaining regions are tags, and used to choose the crop region.

stefanv commented on butterfly/binarization.py:
```diff
@@ -9,60 +9,86 @@
 location = './cachedir'
 memory = Memory(location, verbose=0)
 
+MIN_REGION_HEIGHT = 0.05
```
Please add comments to indicate what these mean.

stefanv commented on butterfly/binarization.py:
```diff
@@ -9,60 +9,86 @@
     ---------
-    binary : 2D array
-        Binarized image of the entire RGB image
+    image_rgb : color image
```
`image_rgb : (M, N, 3) ndarray`

denniscfeng commented:
The test that I changed was failing the old version because I changed the parameters that get passed into find_tags_edge from a binary image to a 3d image, so I had to change the test to reflect that. It is passing the new version of the test.
